tests/integration_tests_control_files/reduced_order/CMakeLists.txt

```diff
@@ -39,9 +39,24 @@ add_test(
         WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
 )
 
+#configure_file(1d_burgers_rewienski_testing.prm 1d_burgers_rewienski_testing.prm COPYONLY)
 #add_test(
 #        NAME 1D_BURGERS_REWIENSKI_TESTING
-#        COMMAND mpirun -np 1 ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_1D -i ${CMAKE_CURRENT_BINARY_DIR}/1d_burgers_rewienski_pod_adaptation.prm;
+#        COMMAND mpirun -np 1 ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_1D -i ${CMAKE_CURRENT_BINARY_DIR}/1d_burgers_rewienski_testing.prm;
+#        WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
+#)
+
+#configure_file(1d_burgers_rewienski_fd_sensitivity.prm 1d_burgers_rewienski_fd_sensitivity.prm COPYONLY)
+#add_test(
+#        NAME 1D_BURGERS_REWIENSKI_SENSITIVITY
+#        COMMAND mpirun -np 1 ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_1D -i ${CMAKE_CURRENT_BINARY_DIR}/1d_burgers_rewienski_fd_sensitivity.prm
+#        WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
+#)
+
+#configure_file(1d_burgers_viscous_fd_sensitivity.prm 1d_burgers_viscous_fd_sensitivity.prm COPYONLY)
+#add_test(
+#        NAME 1D_BURGERS_VISCOUS_SENSITIVITY
+#        COMMAND mpirun -np 1 ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_1D -i ${CMAKE_CURRENT_BINARY_DIR}/1d_burgers_viscous_fd_sensitivity.prm
```

Comment:
Any reason why they are all commented out?

Reply:
Like I mentioned above, the fd_sensitivity tests are used to generate solution and sensitivity snapshots. They don't really test anything (yet), they just return 0. Eventually, I will probably use them within a cmake test to generate a basis and then perform a test, similar to what is done for the `1D_BURGERS_REWIENSKI_POD_ADAPTATION` and `1D_BURGERS_REWIENSKI_REDUCED_ORDER` tests.